### PR TITLE
Change check for dvrLive in controlbar.js

### PR DESCRIPTION
**Pull request by DanFerrer**
### This PR will...

- Explicitly check if ```dvrLive === false```
- FIx linting: End of line needed in ```events.js```

### Why is this Pull Request needed?

```!dvrLive``` doesnt work because ```dvrLive``` can sometimes be undefined and the live button UI is dependent on it being true or false.

#### Addresses Issue(s):

JW8-2278

### CODE REVIEW

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/js/view/controls/controlbar.js [hunk 1]
```diff
@@ -277,9 +277,9 @@ export default class Controlbar {
         _model.change('dvrLive', (model, dvrLive) => {
             const { liveBroadcast, notLive } = localization;
             const liveElement = this.elements.live.element();
-
-            toggleClass(liveElement, 'jw-dvr-live', !dvrLive);
-            setAttribute(liveElement, 'aria-label', dvrLive ? liveBroadcast : notLive);
+            const dvrLiveFalse = dvrLive === false;
```
Comment (karimJWP) [inline src/js/view/controls/controlbar.js]: rename to `dvrNotLive`

Comment (karimJWP) [reply src/js/view/controls/controlbar.js]: add a comment explaining that 'jw-dvr-live' actually means 'DVR mode but not at the live edge'.

File: src/js/view/controls/controlbar.js [hunk 2]
```diff
@@ -277,9 +277,9 @@ export default class Controlbar {
...
+            toggleClass(liveElement, 'jw-dvr-live', dvrLiveFalse);
+            setAttribute(liveElement, 'aria-label', !dvrLiveFalse ? liveBroadcast : notLive);
```
Comment (karimJWP) [inline src/js/view/controls/controlbar.js]: remove negation for easier readability
`setAttribute(liveElement, 'aria-label', dvrNotLive ? notLive : liveBroadcast);`

**Pull request merged**